packages/tupaia-web/src/features/Dashboard/Dashboard.tsx

```diff
@@ -74,15 +79,16 @@ const Title = styled(Typography)`
   line-height: 1.4;
 `;
 
-const ChartsContainer = styled.div<{
+const DashboardItemsWrapper = styled.div<{
   $isExpanded: boolean;
 }>`
-  display: grid;
+  display: ${({ $isExpanded }) =>
+    $isExpanded
+      ? 'grid'
+      : 'block'}; // when in a column, the items should be stacked vertically. Setting to display: block fixes and issue with the chart not contracting to the correct width
   background-color: ${({ theme }) => theme.panel.secondaryBackground};
-  grid-template-columns: repeat(auto-fill, minmax(${MIN_SIDEBAR_WIDTH}px, auto));
+  grid-template-columns: repeat(${({ $isExpanded }) => ($isExpanded ? 2 : 1)}, 1fr);
```

Comment:
this could probably be simplified since it doesn't apply when isExpanded is false